packages/orchestration/src/examples/stakeBld.contract.js

```diff
@@ -34,33 +41,46 @@ export const start = async (zcf, privateArgs, baggage) => {
     baggage,
     privateArgs.marshaller,
   );
-  const makeLocalchainAccountKit = prepareLocalchainAccountKit(
+  const makeLocalChainAccountKit = prepareLocalChainAccountKit(
     baggage,
     makeRecorderKit,
     zcf,
   );
 
+  // Mocked until #8879
+  // Would expect this to be instantiated elsewhere, and passed in as a reference
+  const agoricChainInfo = prepareMockChainInfo(zone);
+
+  async function makeLocalAccount() {
+    const account = await E(privateArgs.localchain).makeAccount();
+    const address = await E(account).getAddress();
+    return makeLocalChainAccountKit({
+      account,
+      address,
+      storageNode: privateArgs.storageNode,
+      timer: privateArgs.timerService,
+      timerBrand: privateArgs.timerBrand,
+      agoricChainInfo,
+    });
+  }
+
   const publicFacet = zone.exo('StakeBld', undefined, {
     makeStakeBldInvitation() {
       return zcf.makeInvitation(
         async seat => {
           const { give } = seat.getProposal();
           trace('makeStakeBldInvitation', give);
           // XXX type appears local but it's remote
```

Comment:
this comment was for `E(privateArgs.localchain).makeAccount()`. best to remove it at this point